src/fiskaltrust.ifPOS/v1/me/Models/RegisterCashDepositRequest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace fiskaltrust.ifPOS.v1.me
+{
+    [DataContract]
+    public class RegisterCashDepositRequest
+    {
+        /// <summary>
+        /// Message identifier, equal to ftQueueItemId when used by fiskaltrust.
+        /// </summary>
+        [DataMember(Order = 10)]
+        public Guid RequestId { get; set; }
+
+        /// <summary>
+        /// Unique code of the cash register, assigned by the central invoice register (CIS) service when registering a cash register.
+        /// </summary>
+        [DataMember(Order = 20)]
+        public string TcrCode { get; set; }
+
+        /// <summary>
+        /// The moment in which the cash deposit was made to the till.
+        /// </summary>
+        [DataMember(Order = 30)]
+        public DateTime Moment { get; set; }
+
+        /// <summary>
+        /// When not null, signalizes that this receipt was late-signed because of the given reason. Null when the receipt was not processed in late-signing mode.
+        /// </summary>
+        [DataMember(Order = 40)]
+        public SubsequentDeliveryType? SubsequentDeliveryType { get; set; }
+
+        /// <summary>
+        /// Amount that is deposited in the till.
+        /// </summary>
+        [DataMember(Order = 50)]
+        public decimal Amount { get; set; }
```

Comment:
Maybe call this `AbsoluteAmount` to clarify that this should be the absolute amount cash in the register

Reply:
So, this is actually the amount of cash that is available in the cash register after this operation, right?
So e.g. if I already had 10€, and I'm adding 90€ more, this value would be 100€?